ESP32/test/test_embedded/test_sntp.cpp

```diff
@@ -0,0 +1,102 @@
+
+#include "gtest/gtest.h"
+#include "time.h"
+// external libaries
+#include "esp_netif_sntp.h"
+#include "esp_sntp.h"
+// source libaries
+#include "sntp_service.h"
+// test libaries
+#include "build_defs.h"
+#include "test_wifi.h"
+
+TEST_F(WifiTest, Sntp) {
+  sntp_sync_status_t sntp_status;
+  Sntp_service sntp;
+
+  esp_err_t err_connection;
+
+  err_connection = m_wifi_handler.connect_to_wifi();
+
+  EXPECT_EQ(err_connection, ESP_OK)
```

Comment:
explicit definition of _err_connection_ not necessary, could all be done inside expect